ecs.go

```diff
@@ -24,6 +25,9 @@ func (e *ecsClient) describeCluster(cluster string) (*ecs.Cluster, error) {
 	if err != nil {
 		return nil, err
 	}
+	if len(resp.Clusters) == 0 {
+		return nil, fmt.Errorf("Cluster %s not found. Did you specify the correct cluster name for your region?", cluster)
```

Comment:
Looks good! But maybe we can improve the message to say something more explicitly about the region setting.
For example: `Make sure you specify the region of your cluster with --region` ? 
or
`Did you specify the correct --cluster and --region?`
or 
`Please check your --cluster and --region parameters`